.github/ISSUE_TEMPLATE/mismatch-report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: 'Mismatch report'
```

Comment:
I'm guessing most users  will be here filing a bug because of two possible scenarios:

1. "This CVE doesn't apply to my project because that component isn't in there" 
2. "this component listed in my SBOM is incorrect"

They probably will learn the term "mismatch" only after they read the docs, and this might be their entry point to finding them! So maybe change this to cover those scenarios...
 
```suggestion
name: 'Report a false positive or incorrect component (mismatch)'
```